hack/tests/verification-tests.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash -e
+# Get verification tests source code prepare it and run the verification tests (Bushslicer)
+
+set -o pipefail
+
+. hack/tests/ci-prepare.sh
+
+function get_az_hosts() {
+  local IFS=$'\n'
+  for ip in $(az vmss list-instance-public-ips -g $RESOURCEGROUP -n ss-master --query "[].ipAddress" -o tsv) ; do
+    i+=${ip}':master:node,'
+  done
+  echo ${i%?}
+}
+
+BRANCH="${BRANCH:-v3}"
+VERIFICATION_TESTS_GIT="${VERIFICATION_TESTS_GIT:-https://github.com/openshift/verification-tests.git}"
+# Env vars for setting up test
+ARO_PUBLIC_HOSTNAME="${ARO_PUBLIC_HOSTNAME:-$(awk '/^  publicHostname:/{ print $2}' ${HOME}/go/src/github.com/openshift/openshift-azure/_data/containerservice.yaml)}"
+RESOURCEGROUP="${RESOURCEGROUP:-$(awk '/^name:/{ print $2}' ${HOME}/go/src/github.com/openshift/openshift-azure/_data/containerservice.yaml)}"
+export OPENSHIFT_ENV_OSE_WEB_CONSOLE_URL="${OPENSHIFT_ENV_OSE_WEB_CONSOLE_URL:-https://${ARO_PUBLIC_HOSTNAME}/}"
+export BUSHSLICER_CONFIG='{"global": {"browser": "chrome"}}'
+export BUSHSLICER_DEFAULT_ENVIRONMENT="${BUSHSLICER_DEFAULT_ENVIRONMENT:-ose}"
+export OPENSHIFT_ENV_OSE_API_PORT="${OPENSHIFT_ENV_OSE_API_PORT:-443}"
+export OPENSHIFT_ENV_SERVICES_AZURE_HOST_CONNECT_OPS_USER="${OPENSHIFT_ENV_SERVICES_AZURE_HOST_CONNECT_OPS_USER:-cloud-user}"
+export OPENSHIFT_ENV_SERVICES_AZURE_HOST_CONNECT_OPS_SSH_PRIVATE_KEY="${OPENSHIFT_ENV_SERVICES_AZURE_HOST_CONNECT_OPS_SSH_PRIVATE_KEY:-${HOME}/go/src/github.com/openshift/openshift-azure/_data/_out/id_rsa}"
+export OPENSHIFT_ENV_OSE_USER_MANAGER_USERS="${OPENSHIFT_ENV_OSE_USER_MANAGER_USERS:-:MYLOGINTOKEN}"
```

Comment:
where does `MYLOGINTOKEN` come from?

Reply:
At present exactly how we get the login token is still being worked out. The previous job for Bushslicer pulled a username and password from the containerservice file. That user account is no longer available. I intend to have the caller of the script provide the token as one of the environment variables as is the case for most of the other config variables. MYLOGINTOKEN is just a placeholder to hopefully make it easier to see why nothing ran in the event that we never set a login token.